pendulum/parser.py

```diff
@@ -13,7 +14,7 @@
 from .tz import UTC
 
 
-def parse(text, **options):
+def parse(text, **options):  # type: (str, **typing.Any) -> str
```

Comment:
Is the return type really `str`? I am not familiar with the pendulum code base, but this looks suspicious to me.

Reply:
I believe the correct return type is `typing.Union[pendulum.Date, pendulum.Time, pendulum.Duration]`.